ui-v2/app/components/consul-service-instance-list/index.hbs

```diff
@@ -0,0 +1,47 @@
+{{yield}}
+{{#if (gt items.length 0)}}
+  <ListCollection @cellHeight={{73}} @items={{items}} class="consul-service-instance-list" as |item index|>
+    <a href={{href-to routeName item.Service.Service item.Node.Node (or item.Service.ID item.Service.Service)}}>
+      <span>
+        {{item.Service.ID}}
+      </span>
+      <ul>
+        <ConsulExternalSource @item={{item.Service}} as |ExternalSource|>
+          <li>
+            <ExternalSource />
+          </li>
+        </ConsulExternalSource>
+      {{#with (reject-by 'ServiceID' '' item.Checks) as |checks|}}
+        <li class={{service/instance-checks checks}}>
+          {{checks.length}} service checks
+        </li>
+      {{/with}}
+      {{#with (filter-by 'ServiceID' '' item.Checks) as |checks|}}
+        <li class={{service/instance-checks checks}}>
+          {{checks.length}} node checks
+        </li>
+      {{/with}}
+      {{#if (get proxies item.Service.ID)}}
+        <li class="proxy">
+          connected with proxy
+        </li>
+      {{/if}}
+      {{#if (gt item.Node.Node.length 0)}}
+         <li class="node">
+          {{item.Node.Node}}
+        </li>
+      {{/if}}
```

Comment:
FYI, pretty sure all services need to have a Node registered, this doesn't hurt though just incase I'm wrong about that.